datadog/dogshell/monitor.py

```diff
@@ -148,14 +163,29 @@ def _file_post(cls, args):
     def _update(cls, args):
         api._timeout = args.timeout
         format = args.format
-        options = None
+
+        to_update = {}
+        if args.type:
+            to_update['type'] = args.type
+            msg = "[DEPRECATION] `type` is no longer required to `update` and may be omitted"
+            warnings.warn(msg, DeprecationWarning)
+        if args.query:
+            to_update['query'] = args.query
+            msg = "[DEPRECATION] `query` is no longer required to `update` and may be omitted"
+            warnings.warn(msg, DeprecationWarning)
```

Comment:
Where do you use the new optional type and query arguments ? This is only for the deprecated positional ones. I think the parser will assign to the same variables, which is why your test works I assume.

Reply:
Ah rats.  It looks like I lost my updates here when I merged w/master and didn't notice.  Will go back and fix.  Used a different `dest` for the optional ones

Reply:
@zippolyte added this back and tests passed locally.  I didn't add anything to explicitly test this but there are tests that either positional or optional arguments and they're both passing